multitest.sh

```diff
@@ -4,14 +4,9 @@ set -e
 
 gem list --local bundler | grep bundler || gem install bundler --no-ri --no-rdoc
 
-echo 'Running bundle exec rspec spec against activesupport / activerecord 2.3.12...'
+echo 'Running bundle exec rspec spec against activesupport / activerecord 5.1.7...'
 
-AARL_ACTIVERECORD_VERSION=2.3.12 bundle update activerecord
+AARL_ACTIVERECORD_VERSION=5.1.7 bundle update activerecord
```

Comment:
Should we also add a spec for the latest version of 5.2 since we're going to need to upgrade to that anyway?